usaspending_api/accounts/v2/filters/account_download.py

```diff
@@ -157,7 +158,8 @@ def retrieve_fyq_filters(account_type, filters):
         if account_type == 'award_financial':
             reporting_period_start = '{}__gte'.format(reporting_period_start)
             reporting_period_end = '{}__lte'.format(reporting_period_end)
-            start_date = datetime.date(filters['fy']-1, 10, 1)
+            if str(filters['quarter']) != '1':
```

Comment:
What is supposed to happen if `filters['quarter'] == 1`?

Reply:
If it's `quarter` 1, what is returned by `retrieve_fyq_filters` is correct.